package.json

```diff
@@ -10,14 +10,19 @@
     "eslint-plugin"
   ],
   "author": "Chris Sauve <chris.sauve@shopify.com>",
+  "engines": {
+    "node": ">=6.11.1"
+  },
   "scripts": {
     "check": "yarn run lint && yarn test",
     "rules-status": "eslint-index lib/config/all.js  --format table",
     "rules-omitted": "eslint-index lib/config/all.js  --status omitted",
     "lint": "eslint . --max-warnings 0",
     "test": "NODE_PATH=$NODE_PATH:./transforms:./tests mocha 'tests/**/*.js' --reporter spec --compilers js:babel-core/register",
     "test:watch": "yarn test -- --watch --reporter min",
-    "test:cover": "babel-node node_modules/.bin/isparta cover --report text --report html _mocha -- --reporter spec --recursive tests/"
+    "test:cover": "babel-node node_modules/.bin/isparta cover --report text --report html _mocha -- --reporter spec --recursive tests/",
+    "preversion": "yarn run check",
+    "postversion": "git push && git push --tags"
```

Comment:
Yes, they both run when a new version is set, but not when queried. i.e. `npm version`.

From https://docs.npmjs.com/misc/scripts#description:
- preversion: Run BEFORE bumping the package version.
- version: Run AFTER bumping the package version, but BEFORE commit.
- postversion: Run AFTER bumping the package version, and AFTER commit.